website/static/js/scrollspy.js

```diff
@@ -0,0 +1,44 @@
+// Based in part on
+// https://github.com/makotot/scrollspy/blob/master/src/js/modules/scrollspy.js
```

Comment:
I think it is no longer based on it.

Reply:
Maybe "inspired by"?  I could remove the link and just mention that ScrollSpy is a thing.  I also wonder whether this file should be called "scrollspy.js" or something else...